.changeset/cuddly-peas-try.md

```diff
@@ -0,0 +1,15 @@
+---
+"@primer/react-brand": minor
+---
+
+Added a new `size` prop to `Button` and a `medium` sized variant, making the previous `large` size optional.
+
+⚠️ This update should be considered a breaking change, despite being issued in a `minor` release. 
+
+Selectable values: `medium` (default) and `large`.
+
+**Upgrade steps:**
+
+```diff
+- <Button>before</Button>
++ <Button size="large">after</Button>
```

Comment:
missing a closing ``` here. Think it was chopped off my suggestion earlier because I nested code blocks

can you add? I can't seem to add it without MD editor converting it :(

```diff
- <Button>before</Button>
+ <Button size="large">after</Button>
```

Reply:
Should be fixed now